my-app/src/HW37/hw-37.tsx

```diff
@@ -0,0 +1,28 @@
+import { Button } from "./buttonBurger"
+import { Title } from "./title";
+import style from "./buttonBurger.module.css"
+import { Tabs } from "./tabs";
+// import styles from "./tabs.module.css"
+
+export const Hw37 = () => (
+    <div className={style.div}>
+        <Title />
+        <div className={style.buttonDiv}>
+            <Button touched handleClick={() => console.log('OK')}>
+                <div>
+                    <hr className={style.burgerSpan}></hr>
+                    <hr className={style.burgerSpan}></hr>
+                    <hr className={style.burgerSpan}></hr>
+                </div>
```

Comment:
this is part of a button component and should be inside it. Not here
same as closeSpan